tic-tac-toe-app/GameBoard.jsx

```diff
@@ -0,0 +1,86 @@
+const calculateWinner = (board) =>{
+  const winningConditions = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ];
+  
+  for(let i=0; i < winningConditions.length; i++){
+    const [a,b,c] = winningConditions[i];
+    if(board[a] && board[a] === board[b] && board[a] === board[c]){
+      console.log(board);
+      console.log(`The winner is player ${board[a]}`);
+      return board[a];
+  }
+}
+  return null;
+};
+
+
+const GameBoard = () => {
+  const [board, setBoard] = React.useState([]);
+  const [turn, setTurn] = React.useState('O');
+  // first i want to check who turn it is and alternate between the two
+  // this function is responsible for changing the player turns
```

Comment:
The inline comments are helpful for other developers to gain a better understanding of what the code represents. 👍